moped-editor/src/views/projects/projectView/ProjectComponents/ComponentListItem.js

```diff
@@ -63,24 +64,35 @@ export default function ComponentListItem({
         ref={component._ref}
       >
         {isComponentMapped ? Icon : <ErrorOutlineIcon color="error" />}
-        <Box display="flex" alignItems="center" width="100%">
-          <ListItemText
-            className={classes.listItemText}
-            primary={primary}
-            secondary={
-              <>
-                <>{secondary}</>
-                <span className={classes.additionalListItemText}>
-                  {additionalListItemText}
-                </span>
-              </>
-            }
-          />
-          <ListItemSecondaryAction>
-            <IconButton color="primary" onClick={onZoomClick} size="large">
-              <ZoomInIcon />
-            </IconButton>
-          </ListItemSecondaryAction>
+        <Box>
+          <Box display="flex" alignItems="center" width="100%">
+            <ListItemText
+              className={classes.listItemText}
+              primary={primary}
+              secondary={
+                <>
+                  <>{secondary}</>
+                  <span className={classes.additionalListItemText}>
+                    {additionalListItemText}
+                  </span>
+                </>
+              }
+            />
+            <ListItemSecondaryAction>
+              <IconButton color="primary" onClick={onZoomClick} size="large">
+                <ZoomInIcon />
+              </IconButton>
+            </ListItemSecondaryAction>
+          </Box>
+          {!!component.moped_phase && (
+            <Box width="100%" sx={isExpanded ? { pt: 0.5 } : { py: 0.5 }}>
```

Comment:
@chiaberry I think that spacing should remain constant whether expanded or not since I am seeing the component geometry type icon shift when expanding/collapsing. could you try out `my: 0.5` and `ml: 1` to make the badge aligned with the component type text above it and also keep the bottom spacing static instead of the dynamic padding? I'm on the fence about adding margin top. Maybe that is too much white space?

excuse my IASIP Charlie conspiracy arrows in this screenshot 🙃
<img width="348" alt="Screenshot 2024-11-05 at 11 40 26 AM" src="https://github.com/user-attachments/assets/4426c43a-58e6-42df-8017-8962cfbcc0c8">

Also wanted to note that this is another TIL that these integer values map to the `theme.spacing` values when used inside of the `sx` prop. Nice! 📚

Reply:
more discussion [in this Slack thread](https://austininnovation.slack.com/archives/CNUEPKLB1/p1730831271900659)